deftree/svcparse/parser.go

```diff
@@ -360,29 +504,54 @@ func ParseBindingFields(lex *SvcLexer) ([]*Field, error) {
 			lex.UnGetToken()
 			break
 		}
+		// Use recursion to parse custom HTTP verb sections
+		if val == "custom" {
+			err := fastForwardTill(lex, "{")
+			if err != nil {
+				return nil, nil, err
+			}
+			c, _, err := ParseBindingFields(lex)
```

Comment:
Can you not have custom within custom? Not sure of the significance of ignoring the middle param.

Reply:
No, you cannot have a custom within a custom, which is why I underscore away what would be the "custom" fields (the middle param).